src/history/HistoryArchiveManager.cpp

```diff
@@ -156,6 +158,31 @@ HistoryArchiveManager::selectRandomReadableHistoryArchive() const
     }
 }
 
+void
+HistoryArchiveManager::logReportOnArchiveConfig() const
+{
+    auto& ws = mApp.getWorkScheduler();
+    for (auto const& archive : mArchives)
```

Comment:
I don't think you want to do reporting on the archives we publish to. So I think we want read only archives (there is already logic that computes those in `checkSensibleConfig`, so perhaps we just need to make it a member variable)

Reply:
It's fine to loop over all archives: people can still make typos in archives they publish to (this functionality is to help detect problems early, and not just when we need archives)